mpsl/CHANGELOG.rst

```diff
@@ -71,7 +91,7 @@ Changes
 Bug fixes
 =========
 
-* Fixed an issue where the high frequency clock and ``TIMER0`` were not turned off during idle periods shorter than 9 ms (DRGN-14152).
+* Fixed an issue where both the High-Frequency Clock and ``TIMER0`` were not turned off during idle periods shorter than 9 ms (DRGN-14152).
```

Comment:
We should be consistent with how we denote the high and low frequency clock. See  fixed issue for DRGN-15176
```suggestion
* Fixed an issue where both the high-frequency Clock and ``TIMER0`` were not turned off during idle periods shorter than 9 ms (DRGN-14152).
```

Reply:
fixed